docs/deployments.md

```diff
@@ -8,41 +8,37 @@ Cloud Foundary scripts exist within each application in the /CF directory. These
 
 Migrations are manual.
 
-1. [get access to a rails console on the box](/docs/infrastructure)
+1. [get access to a rails console on the box](infrastructure.md)
 1. run `bin/rails db:migrate`
 1. run `cf v3-zdt-restart ccs-rmi-api-staging` for these changes to take effect
 
+## CHANGELOG
+
+As outlined in the [DSD code management workflow guide], Release/* branches are protected now so we can no longer push directly to them. To avoid having to make another pull request just for CHANGELOG additions at the end, it’s may be more efficient to update it as we go in Feature/* branches.
+
+In CHANGELOG.md:
+  - document the change your feature contributes to the release in a bullet point form
+  - Make sure there’s a corresponding link to the diff at the bottom of the file
+
 ## Staging
 
 Deployments to staging are automatic when new changes are pushed via
 pull request to the develop branch.
 
 ## Production
 
-As outlined in the [dxw development workflow guide], production deploys are
-done by manually merging `develop` into `master`. To give us a slightly more
-formal process around what gets deployed and when and also to give us
-visibility into the things that have been deployed, we additionally follow
-these steps when releasing to production:
-
-### 1. Create a release branch and make a pull request
+Having tested in staging, reviewed and merged the relevant Feature/* branches back into develop: 
 
-  - Create a branch from `develop` for the release called `release-X` where X is the release
-    number
-  - Update [CHANGELOG.md](CHANGELOG.md) to:
-     - document the changes in this release in a bullet point form
-     - add a link to the diff at the bottom of the file
-  - Document the changes in the commit message as well
-  - `git push <your-branch-name>`, e.g. `git push release-45`
-  - Create a tag for the release in the format `release-X`
-  - Create a pull request for the release with content from the `CHANGELOG.md`
-  - Get that pull request reviewed and approved
+### 1. Create a Release/* branch
 
-### 2. Review and merge the release pull request
+  - Create a branch from `develop` for the release called `Release/X` where X is the release number
+  - Test the release cut in the preprod environment.
```

Comment:
Feel like there'll be a step between these two right?

Reply:
You mean Pre-Prod step?

Reply:
A step for pushing into the preprod environment, however that'll happen